master/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#
+# Triton-optimized Mesos master
+#
+FROM 		misterbisson/triton-mesos:2015-07-06-triton
```

Comment:
There's a huge opportunity here to significantly reduce the size of these images by extracting the binaries from `misterbisson/triton-mesos` and injecting them into a minimal container as @tgross has demonstrated. I'm leaving that for a future revision.